bin/redshift/local_delete_aws_redshift.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+cd "${DIR}/../.." || exit 1
+
+env AWS_REGION=us-east-2 \
```

Comment:
why us-east-2?

this is another reason not to have `local_....*sh` version of the scripts.
if someone wants to use the scripts, they can just fire up the very same scripts CI uses.
the only required things for that is setting the variables CI would set (via secrets/vars), which is not hard.

picking default values that work for you or me doesn't make them work for everyone